wallet-new/server/Cardano/Wallet/API/V1/Swagger.hs

```diff
@@ -353,42 +384,52 @@ Since you have, for now, only a single wallet, you'll see something like this:
 }
 ```
 
-All the wallet's accounts are listed under the `addresses` field. You can communicate one of these addresses to receive _Ada_ on the associated account.
+All the wallet's accounts are listed under the `addresses` field. You can communicate one of
+these addresses to receive _ADA_ on the associated account.
 
 
-Sending Money
--------------
+Sending ADA
+-----------
 
-In order to send _Ada_ from one of your accounts to another address, you must create a new payment transaction using the `POST /api/v1/transactions` endpoint:
+In order to send _ADA_ from one of your accounts to another address, you must create a new
+payment transaction using the [`POST /api/v1/transactions`](#tag/Transactions%2Fpaths%2F~1api~1v1~1transactions%2Fpost)
+endpoint as follows:
 
 ```
 curl -X POST https://localhost:8090/api/v1/transactions \
-     -H "Content-Type: application/json; charset=utf-8" \
-     -H "Accept: application/json; charset=utf-8"       \
-     --cacert ./scripts/tls-files/ca.crt                \
-     -d '{                                              \
-  "destinations": [{                                    \
-    "amount": 14,                                       \
-    "address": "A7k5bz1QR2...Tx561NNmfF"                \
-  }],                                                   \
-  "source": {                                           \
-    "accountIndex": 0,                                  \
-    "walletId": "Ae2tdPwUPE...8V3AVTnqGZ"               \
-  }                                                     \
+  -H "Accept: application/json; charset=utf-8" \
+  -H "Content-Type: application/json; charset=utf-8" \
+  --cacert ./scripts/tls-files/ca.crt \
+  -d '{
+  "destinations": [{
+    "amount": 14,
+    "address": "A7k5bz1QR2...Tx561NNmfF"
+  }],
+  "source": {
+    "accountIndex": 0,
+    "walletId": "Ae2tdPwUPE...8V3AVTnqGZ"
+  },
+  "spendingPassword": "5416b2988745725998907addf4613c9b0764f04959030e1b81c603b920a115d0"
 }'
 ```
 
-Note that, in order to perform a transaction, you need to have enough coins on the source account! The Cardano API is designed to accomodate multiple recipients payments out-of-the-box; notice how `destinations` is a list of addresses (and corresponding amounts).
+Note that, in order to perform a transaction, you need to have enough existing _ADA_ on the
+source account! The Cardano API is designed to accomodate multiple recipients payments
+out-of-the-box; notice how `destinations` is a list of addresses (and corresponding amounts).
 
-When the transaction succeeds, funds are no longer available in the sources addresses, and are soon made available to the destinations. Note that, you can at any time see the status of your wallets by using the `GET /api/v1/transactions/{{walletId}}` endpoint:
+When the transaction succeeds, funds are no longer available in the sources addresses, and are
+soon made available to the destinations in a short delay. Note that, you can at any time see
```

Comment:
"destinations in a short delay" -> "destinations with a short delay"

Reply:
Hmmm... I kinda disagree with that one. Not sure why, but `in` sounds a bit more accurate here imo.

Reply:
What about `within a short delay`?

Reply:
Ahh, I can't quite say *why* `in` feels wrong to me!

> X will happen in an hour

this feels right

> X will happen with an hour

this feels wrong

> X will happen in a delay

this feels wrong

> X will happen with a delay

this feels right

Hmm, I think it boils down to that `X` will happen after a delay, not during the delay. X can't happen *during* the delay, because if X is happening, then the delay is complete. 

After rereading this, `after` seems more appropriate than `with` or `in` anyway